First Draft of Updates to Element Voiding CTs

AlexBrad1eyCT opened the pull request:
Fixes #214 

voiding converted to group and CTs created for earthworks cut, openings and voiding features

AlexBrad1eyCT committed:
First Draft of Updates to Element Voiding CTs

voiding converted to group and CTs created for earthworks cut, openings and voiding features

pjanck commented on IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:
```diff
@@ -0,0 +1,3 @@
+Earthworks Cuttings represent a voiding of an existing or constructed Earthen element (referred to as the voided element)  usually represented by a subtype of _IfcGeotechnicalElement_ or an _IfcEarthworksFill_, t. the _IfcEarthworksCut_ semantically represents the work of removing/excavating material from the voided element, and the associated geometrical representation is the resulting void crea ...
```
What's a `t.`?

AlexBrad1eyCT replied:
oo ehh random sorted.

pjanck commented on IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:
```diff
@@ -0,0 +1,3 @@
+Earthworks Cuttings represent a voiding of an existing or constructed Earthen element (referred to as the voided element)  usually represented by a subtype of _IfcGeotechnicalElement_ or an _IfcEarthworksFill_, t. the _IfcEarthworksCut_ semantically represents the work of removing/excavating material from the voided element, and the associated geometrical representation is the resulting void crea ...
```
You may want to revise `A_IfcEarthworksCut_ May`

AlexBrad1eyCT replied:
sorted

pjanck commented on IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:
```diff
@@ -0,0 +1,3 @@
+Earthworks Cuttings represent a voiding of an existing or constructed Earthen element (referred to as the voided element)  usually represented by a subtype of _IfcGeotechnicalElement_ or an _IfcEarthworksFill_, t. the _IfcEarthworksCut_ semantically represents the work of removing/excavating material from the voided element, and the associated geometrical representation is the resulting void crea ...
+
+The resulting geometry of the relationship when applied is the result of CSG operations where the IfcEarthworksCut representation is subtracted from the voided element representation.
```
```suggestion
The resulting geometry of the relationship, when applied, is the result of CSG operations where the _IfcEarthworksCut_ representation is subtracted from the voided element representation.
```

AlexBrad1eyCT replied:
thanks for catching that one!

pjanck reviewed (changes requested):
Missing on all entries:  `Identification`.

AlexBrad1eyCT committed:
Update Earthworks Cuttings Documentation

AlexBrad1eyCT committed:
Update IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md

Co-authored-by: Stefan Jaud <59165496+pjanck@users.noreply.github.com>

pjanck reviewed (approved):
lftm

AlexBrad1eyCT commented:
@LeeGregory12d, @jmirtsch as providers of Earthworks unit tests your review would be appreciated!

AlexBrad1eyCT committed:
Missing Identifications 

Missing Identification on properties and an error on one of the templates that was not noticed
